public/scss/components/PopUpVoucherCoupon.module.scss

```
@@ -239,18 +278,64 @@
   color: $color_line_black;
 }
 
+.voucherDetailViewDetails
+{
+  @extend .btn;
+  padding: 0;
+  border: none;
+  background-color: transparent;
+  color: $color_blue;
+  @include ExtraSmallReguler;
+  @include absolute(8px, 12px);
```

bisa ditaruh di bawahnya @extend mas
```suggestion
  @extend .btn
  @include ExtraSmallReguler;
  @include absolute(8px, 12px);
```